pkg/storage/etcd/testing/utils.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package etcd
+package testing
```

Comment:
Maybe name "etcdtest" (like ["httptest"](http://godoc.org/net/http/httptest)) to avoid always name colliding with "testing" import.